store/types/validity.go

```diff
@@ -1,15 +1,30 @@
 package types
 
-// AssertValidKey checks if the key is valid(key is not nil)
+import math "math"
+
+const (
+	// 32K
+	MaxKeyLength = math.MaxUint16
+	// 4G
+	MaxValueLength = math.MaxUint32
+)
+
+// AssertValidKey checks if the key is valid(key is not nil and within length limit)
 func AssertValidKey(key []byte) {
 	if len(key) == 0 {
 		panic("key is nil")
```

Comment:
While you are here, yould you fix the panic message and docs? Both nil and empty are disallowed.

Reply:
done